src/histolab/tiler.py

```diff
@@ -667,9 +682,8 @@ def extract(self, slide: Slide, report_path: str = None):
                 f"{slide.level_dimensions(self.level)} at level {self.level}"
             )
 
-        highest_score_tiles, highest_scaled_score_tiles = self._highest_score_tiles(
-            slide
-        )
+        highest_score_tiles = self._highest_score_tiles(slide)
+        highest_scaled_score_tiles = self._highest_score_tiles(slide, scaled=True)
```

Comment:
you need `highest_scaled_score_tiles ` only if you want to save the report, i.e., if `report_path ` is None